classes/Search.php

```diff
@@ -383,6 +383,20 @@ public static function getTags($db, $id_product, $id_lang)
      * @param int $id_lang
      * @return string
      */
+    public static function getSecondaryCategories($db, $id_product, $id_lang)
+    {
+        $categoriesArray = $db->executeS(sprintf('
```

Comment:
It's sometimes clearer and easier to optionally quote this.
I could change that if you prefer a the line to look like:
> executeS(" xxx " .  _DB_PREFIX_ , " xxx {$id_lang} ...");